File: mall-manager-web/src/main/webapp/js/service/brandService.js
 //自定义服务
	    app.service('brandService',function($http){
	    	
	    	this.selectBrandList = function(){
	    		return $http.get('../brand/selectBrandList.do');
	    	}
	    	
	    	this.findAll = function(){
	    		return $http.get('../brand/allBrand.do');
	    	}
	    	
	    	this.findPage = function(page,rows){
	    		return $http.get('../brand/findPage.do?page='+page+'&rows='+rows);
	    	}
	    	
	    	this.add = function(entity){
	    		return $http.post('../brand/add.do',entity);
	    	}
	    	this.update = function(entity){
	    		return $http.post('../brand/update.do',entity);
	    	}
	    	
	    	this.findOne = function(id){
	    		return $http.get('../brand/findOne.do?id='+id);
	    	}
	    	this.dele = function(ids){
	    		return  $http.get('../brand/delete.do?ids='+ids);
	    	}
	    	this.search = function(page,rows,searchEntity){
	    		return $http.post('../brand/search.do?page='+page+'&rows='+rows,searchEntity);
	    	}
	    });

File: mall-sellergoods-interface/src/main/java/com/cblue/sellergoods/service/BrandService.java
package com.cblue.sellergoods.service;

import java.util.List;
import java.util.Map;

import com.cblue.pojo.TbBrand;

import entity.PageResult;

public interface BrandService {
	
	/**
	 * select2获得品牌数据
	 */
	public List<Map> selectBrandList();
	
	/**
	 * 查询所有的品牌
	 */
	public List<TbBrand> getAllBrand();
	/**
	 * 分页查询
	 */
	public PageResult findPage(int pageNum,int pageSize);
	
	/**
	 * 条件分页查询
	 * @param pageNum 当前页码
	 * @param pageSize 每页记录数
	 */
	public PageResult findPage(TbBrand brand, int pageNum,int pageSize);
	
	/**
	 * 增加
	 */
	public void add(TbBrand brand);
	/**
	 * 根据ID获取实体
	 */
	public TbBrand findOne(Long id);	
    /**
	 * 修改
	 */
	public void update(TbBrand brand);
	
	/**
	 * 批量删除
	 */
	public void delete(Long [] ids);
	

}
